woocommerce/wc-admin
# Make products and rates endpoints return zero-value items

Part of #1668.

Makes `products` and `rates` endpoints return zero-value items.

I wanted to create this PR before working on the other endpoints to get some early feedback.

### Screenshots
_(Notice _Beanie_ appears in the table even though its values are 0)_
![image](https://user-images.githubusercontent.com/3616980/53639651-192a3400-3c2a-11e9-9eb5-8c4463e518cd.png)

### Detailed test instructions:
- Go to the _Products_ report.
- Try sorting by every column.
- Start a _Product Comparison_ making sure one of the products has all values to 0 (it has not been sold during the selected date range).
- Verify the table shows all compared products, even the one that has all values to 0.
- Try sorting by every column and verify the 0-value product is correctly sorted.
- Repeat the same with different filters, with an active table search, etc.
- Now, repeat the same in the _Taxes_ report.
